production/grafanacloud-install.sh

```diff
@@ -36,6 +36,9 @@ GCLOUD_API_KEY=${GCLOUD_API_KEY:=}   # API key to communicate to the integration
 # OPTIONAL environment variables.
 #
 
+# Architecture to install.
```

Comment:
Is / should this be mentioned in the associated documentation for running the install script?

Reply:
Nope, the UI will generate the script for you and will have an architecture selector. We don't actually document this script anywhere.